native/cocos/bindings/jswrapper/v8/ScriptEngine.h

```diff
@@ -422,6 +432,7 @@ class ScriptEngine final {
     bool _isWaitForConnect;
 
     uint32_t _vmId;
+    uint32_t _blockingTimeout{1000};
```

Comment:
What's the unit? Please add comment for it.

Reply:
This is a unsued field, it will be removed

Reply:
Removed